resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
     // Common
     'actions'     => 'Actions',
     'airport'     => 'Aéroport',
-    'all'         => 'Voir Tous',
+    'all'         => 'Afficher tout',
     'altn'        => 'Altn',
     'alternate'   => 'Alternatif',
     'close'       => 'Fermer',
-    'darkmode'    => 'Mode Sombre',
+    'darkmode'    => 'Mode Nuit',
     'dest'        => 'Dest',
     'destination' => 'Destination',
-    'fuel_ob'     => 'Carburant à Bord',
-    'latest'      => 'Derniers',
+    'fuel_ob'     => 'Fuel embarqué',
+    'latest'      => 'Récents',
     'map'         => 'Carte',
-    'notes'       => 'Remarques',
     'orig'        => 'Orig',
     'origin'      => 'Origine',
     'registered'  => 'Enregistré',
-    'show_hide'   => 'Voir / Cacher',
+    'show_hide'   => 'Carte / Liste',
```

Comment:
Since "Carte" means "Map" and "Liste" means "List/Table", i think this is good for the livemap but not for the whole context while "Afficher/Cacher is the literal translation of "Show/Hide" and seems more appropriate for all contexts